ocs_ci/ocs/ui/page_objects/virtualmachine_ui.py

```
@@ -432,6 +480,89 @@ def check_grace_period_and_confirm_delete(self):
+    def get_clone_vm_name(self):
+        """
+        Read the pre-filled VM name from the Clone VirtualMachine popup.
+
+        Returns:
+            str: The clone VM name shown in the Name field.
+        """
+        name_input = self.vm_loc["clone_vm_name_input"]
+        wait_for_element_to_be_visible(locator=name_input, timeout=20)
+        el = self.driver.find_element(name_input[1], name_input[0])
+        clone_name = el.get_attribute("value") or el.text.strip()
+        logger.info(f"Clone VM name from popup: '{clone_name}'")
```

Ruff (0.16.0)

[warning] Logging statement uses f-string

(G004)